config.yaml

```diff
@@ -27,7 +27,7 @@ params:
   grpc_vers:
     core: v1.42.0
     go: v1.41.0
-    java: v1.42.1
+    java: v1.43.1
```

Comment:
Already outdated? Shouldn't this be bumped up to 1.44.0 now?

Reply:
It's been outdated several times...